package.json

```diff
@@ -76,8 +77,14 @@
     "typescript": "4.0.2"
   },
   "dependencies": {
+    "find-git-root": "^1.0.4",
+    "meow": "^7.1.0",
+    "mkdirp": "^1.0.4",
```

Comment:
This should be dev dependency to replace `mkdir -p target/es5 ...` with `mkdirp target/es5`.
Since nodejs 10.12.0 `fs.mkdir` accepts a `{ recursive: true }`.

Reply:
+